### ISSUE TITLE
win 11 - new updates with the new exploerpatcher version

### ISSUE THREAD

**Issue by romafrogs** [valinet/ExplorerPatcher#2009]
If im installing the new (2 weeks ago) patcher version

Can i install ALL the new win 11 updates? including preview?

**Comment by Poopooracoocoo** [comment]
No. As per the wiki, EP explicitly does not support Insider builds and preview updates. And currently it does not work properly on the latest GA release of Windows 11, OS Build 22621.2134. If you're already on this build, see if you can uninstall KB5029263. The latest version of EP only claims support up to OS Build 22621.1992.
